charts/cdf/tekton-pipeline/defaults.yaml

```diff
@@ -1,3 +1,3 @@
 gitUrl: https://github.com/cdfoundation/tekton-helm-chart
 namespace: tekton-pipelines
-version: 0.29.0
+version: 0.2.0
```

Comment:
Is this required for this change? I would prefer to keep it in different PRs.

Reply:
Yes.
"Since kpt is very strict about the yaml syntax there is still one more fix needed for kpt to work"
Without cdfoundation/tekton-helm-chart#28 I got problems with duplicate entries in a configmap due to https://github.com/jenkins-x/jx3-versions/blob/master/charts/cdf/tekton-pipeline/values.yaml.gotmpl#L25-L27

Of course I could downgrade the tekton version in cdfoundation/tekton-helm-chart to get a release with my fix but still tekton version 0.29.

Reply:
Ah yes, I remember. ok we should first get 0.1.0 out and then do this IMO.